.github/workflows/devcontainer.yml

```
@@ -1,47 +1,104 @@
     runs-on: ubuntu-latest
-    outputs:
-      image_name: ${{ steps.set_image_name.outputs.image_name }}
     steps:
-      - name: Set Image Name
-        id: set_image_name
+      - name: Checkout repository
+        uses: actions/checkout@v6
+
+      - name: Set registry
+        id: registry
         run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:5:30: Double quote to prevent globbing and word splitting

(shellcheck)